composer.json

```diff
@@ -5,7 +5,7 @@
         "OSL-3.0"
     ],
     "require": {
-        "php": "^7.2|~8.0.0|~8.1.0"
+        "php": "^7.2|~8.0.0|~8.1.0|~8.2.0"
```

Comment:
Didnt we want to leave out PHP requirements in favor of Magento core requirements all together?

Reply:
Yeah, that's probably a better idea.